renovate.json

```
@@ -2,6 +2,5 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "github>conforma/.github//config/renovate/renovate.json"
-  ],
-  "baseBranches": ["main", "release-v0.5", "release-v0.6"]
+  ]
```

I expect it would work the same, but consider this:

```suggestion
    "baseBranches": ["main"]
  ]
```

---

From support it sounds like we shouldn't have any base branches.